TransitionsApp/SecondViewController.swift

```diff
@@ -9,34 +9,53 @@ import Foundation
 import UIKit
 
 final class SecondViewController: UIViewController {
-    @IBOutlet private weak var colorButton: UIButton!
+    @IBOutlet private weak var redButton: UIButton!
     @IBOutlet private weak var closeButton: UIButton!
-    
-    var nameButton = "カラー"
+    @IBOutlet weak var greenButton: UIButton!
+    @IBOutlet weak var blueButton: UIButton!
+    var redColorButton = "赤"
     var buttonName = "戻る"
+    var blueColorButton = "青"
+    var greenColorButton = "緑"
     
     override func viewDidLoad() {
         super.viewDidLoad()
         secondchangeNameButton()
         thirdchangeNameButton()
-        
+        forcecangeNameButton()
+        fifthchangeNameButton()
+        let selectedColor: colour = .blue
+        self.view.backgroundColor = selectedColor.colorSetting
     }
     func secondchangeNameButton(){
-        colorButton.setTitle(nameButton, for: .normal)
+        redButton.setTitle(redColorButton, for: .normal)
```

Comment:
`secondchangeNameButton`の名前だと、第三者から見たらわかりずらいので、赤ボタンなら
```suggestion 
func  redChangeNameButton() {
```
にするかな！
細かくて申し訳ないけど、()の後はスペース空けよう！